test/test_sbol2_sbol3_direct.py

```diff
@@ -244,6 +253,82 @@ def test_seq_componentDefinition(self):
                 tmp2 = Path(tmpdir) / 'doc2_loop.xml'
                 doc2_loop.write(tmp2)
                 self.assertFalse(file_diff(str(tmp2), str(TEST_FILES / 'seq_componentDefinition.xml')))
+
+    def handle_2to3_conversion(self, test_filename, rubric_filename):
```

Comment:
Suggest adding typing to function parameters, both here and elsewhere:
```suggestion
    def handle_2to3_conversion(self, test_filename: str, rubric_filename: str):
```